src/mesh/replicated_mesh.C

```diff
@@ -1476,7 +1478,7 @@ ReplicatedMesh::get_disconnected_subdomains(std::vector<subdomain_id_type> * sub
     representative_elem_ids.push_back(min_id);
     subdomain_counter++;
   }
-  while (visited != n_elem());
+  while (visited != n_active_elem());
```

Comment:
`n_active_elem()` is a relatively expensive (O(N. elem)) function compared to `n_elem()` so it would be best to avoid calling it at every loop iteration. Since I don't think the number of active elements in the mesh changes during this `do` loop, it would be better to compute this value once beforehand.

Reply:
Good to know. Thought it is just a getter to a member variable... I can submit a further fix if desired.

Reply:
I already have one ready. Github went down for some reason, but I will submit it shortly.

Reply:
Great!